stream-chat-android-ui-components/src/main/res/layout/stream_ui_dialog_message_options.xml

```diff
@@ -22,7 +22,7 @@
             android:layout_marginHorizontal="@dimen/stream_ui_spacing_small"
             />
 
-        <FrameLayout
+        <io.getstream.chat.android.ui.common.internal.TouchInterceptingFrameLayout
             android:id="@+id/messageContainer"
```

Comment:
This View ensures that all the touch events on the message bubble are consumed. And, for example, long click on the message doesn't spawn a new message options overlay.